ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -218,44 +227,239 @@ const BackArrow = withCircleBackground(() => (
   <Arrow style={{ height: 21, width: 18 }} />
 ));
 
-interface CardProps {
-  step: number;
+export const RenewalCardBody = (props: {
   moduleStatus: AccessModuleStatus;
-  style: React.CSSProperties;
-  children: string | React.ReactNode;
-}
-const RenewalCard = withStyle(renewalStyle.card)(
-  ({ step, moduleStatus, style, children }: CardProps) => {
-    const { AARTitleComponent } = getAccessModuleConfig(
-      moduleStatus.moduleName
-    );
-    const { lastConfirmedDate, nextReviewDate } =
-      computeRenewalDisplayDates(moduleStatus);
-    return (
-      <FlexColumn style={style}>
-        <div style={renewalStyle.h3}>STEP {step}</div>
-        <div style={renewalStyle.h3}>
-          <AARTitleComponent />
-        </div>
-        <div
-          style={{
-            color: colors.primary,
-            margin: '0.5rem 0',
-            display: 'grid',
-            columnGap: '1rem',
-            gridTemplateColumns: 'auto 1fr',
-          }}
-        >
+  setLoading: (boolean) => void;
+  hide?: boolean;
+  textStyle?: CSSProperties;
+  showTimeEstimate?: boolean;
+}) => {
+  const {
+    moduleStatus,
+    setLoading,
+    hide,
+    textStyle,
+    showTimeEstimate = false,
+  } = props;
+
+  const [, navigateByUrl] = useNavigation();
```

Comment:
Is this a typo?

Reply:
This means we are dereferencing the two elements of the array returned by `useNavigation()` but only using one of them, `navigateByUrl`

Reply:
Fair enough. Thank you.